content/v1.12/getting-started/provider-azure-part-2.md

```diff
@@ -0,0 +1,1050 @@
+---
+title: Azure Quickstart Part 2
+weight: 120
+tocHidden: true
+---
+
+{{< hint "important" >}}
+This guide is part 2 of a series. Follow **[part 1]({{<ref "provider-azure" >}})** 
+to install Crossplane and connect your Kubernetes cluster to Azure.
+
+**[Part 3]({{<ref "provider-azure-part-3">}})** covers patching _CompositeResources_
+and using Crossplane _Packages_.
+{{< /hint >}}
+
+This section creates a _[Composition](#create-a-composition)_, 
+_[CompositeResourceDefinition](#define-a-composite-resource)_ and a
+_[Claim](#create-a-claim)_
+to create a custom Kubernetes API to create Azure resources. This custom API 
+is a _composite resource_ (XR) API.
```

Comment:
I say no, just because I don't want to introduce too much new material at once. Going to the concepts docs is sending them to the deep end. I'd rather have them walk through this to learn what an XR and Claim are at the 101 level and then read the Concepts later to get more details.